Calcula la exponencial de un numero

blancogerardo94 opened the pull request:
Esta funcion recibe un float, calcula la exponencial y regresa el valor de la exponencial de tipo float.

Commit:
Calcula la exponencial de un numero

Esta funcion recibe un float, calcula la exponencial y regresa el valor de la exponencial de tipo float.

charis38 commented on exp.c:
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#include <math.h>
+
+float expo(float num)
```
Que recibe?

charis38 commented on exp.c:
```diff
@@ -0,0 +1,11 @@
+{
+  float res;
+
+  res = expf(num);
```
Tienes que usar serie de taylor

Commit:
Calcula la exponencial mediante la serie de Taylor

Se modifico la funcion para calcular la exponencial mediante la serie de taylor con 30 iteraciones.
Se podria quitar el include de math.h si es necesario.

blancogerardo94 commented:
Creo que ya jala bien esto

blancogerardo94 commented on exp.c:
```diff
@@ -3,9 +3,26 @@
 {
-  float res;
-
-  res = expf(num);
```
Se me olvido quitar esta linea

blancogerardo94 replied:
no entiendo porque sigue ahi esa linea

eortizl commented on exp.c:
```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
```
comentarios tipo Doxygen
